### ISSUE TITLE
Facebook Messsenger "Could not display conversation list"

### ISSUE THREAD

**Issue by plazia** [brookhong/Surfingkeys#962]
## Prelude

I could not find any information about this in the FAQ. I have searched for issues on GitHub and there is an issue similar though I don't think it's the same. The similar issue is regarding the Facebook Messenger input box. This is the contact list on the left hand side of the website https://www.messenger.com

## Error details

SurfingKeys: <0.9.44>

Browser: Firefox 66.0.5-1 Arch Linux (Rolling)

URL: https://www.messenger.com/

Add-on enabled: https://i.imgur.com/WoX9QVd.png
Add-on disabled: https://i.imgur.com/zFHfwwi.png

When you load the page the contact list on the left hand side is absent and a message saying "Could not display conversation list" appears. If you disable Surfingkeys add-on in Firefox the problem goes away. 

No action is required to reproduce this problem you simply need to load the page and it occurs. I was able to narrow it down to Surfingkeys add on simply by eliminating all other add-ons. Well initially I started Firefox in --safe-mode and found that the error was not present. Then I started enabling add-ons one-by-one. Sure enough the problem is with Surfingkeys which is unfortunate because since I discovered it, it's become my favorite and practically indispensable add-on.

## Context

I use Surfingkeys on a daily basis with Firefox for its easy to use vim navigation.

## Edit

This also affects Facebook Marketplace in Firefox. https://apps.facebook.com/marketplace/ If you go to the URL with Surfingkeys enabled, 90 % of the time it refuses to load content on the page. Disable Surfingkeys and it works 100 % of the time.

**Comment by fweth** [comment]
Interestingly, disabling Surfingkeys for messenger.com also didn't work for me (it displays the yellow banner that SK is disabled for messenger.com but the conversation list still doesn't load, even after hard refresh).

https://whatsmybrowser.org/b/EKNTJFG

**Comment by fweth** [comment]
OK, as a workaround, I now first disable SK in messenger.com, then disable it globally in the add-ons menu, then reload messenger.com and then enable SK globally again.  This only works when SK is also locally disabled since otherwise it would automatically refresh the page.

**Comment by rstacruz** [comment]
Confirming that it happens here too... Another Firefox + Arch Linux user here.

**Comment by Pete2030** [comment]
Unfortunately, because I was keen to use the add-on/extension, I found, by process of elimination, that "Translation Comparison by Smart Link Corporation" with the tri-fold icon, caused the blockage on my system. I tried loading and reloading Messenger, after changing each option, but without success. 

I would be very pleased if a workaround could be found and made available for this extension. Unfortunately, I am less than a novice at coding and programming, so can not offer any ideas. 

Cheers, Pete_
